app/models.py

```diff
@@ -26,6 +27,9 @@ def insert_roles():
             'User': (
                 Permission.GENERAL, 'main', True
             ),
+            'AgencyWorker': (
+                Permission.WORKER, 'main', True
```

Comment:
With regards to `True` (aka the third element in the tuple): what does this mean? I know it sets the `default` field to True. What does this mean?

I hypothesize that it means that users are, by default, given the permission with `default` of True. Thus, if there are several permissions with `default` of True, this won't work.

Can you check how it works?